servers/rendering/renderer_rd/shaders/environment/sdfgi_debug.glsl

```diff
@@ -37,16 +37,13 @@ layout(push_constant, std430) uniform Params {
 	uint max_cascades;
 
 	ivec2 screen_size;
-	bool use_occlusion;
 	float y_mult;
 
-	int probe_axis_size;
 	float z_near;
-	float reserved1;
-	float reserved2;
 
-	mat4 cam_transform;
-	mat4 inv_projection;
+	float inv_projection[4][3];
```

Comment:
This could be a mat3x4 which I beleive can be promoted to a mat4 with just a constructor like ``mat4(mat3x4_param)``

Reply:
Likewise.

Reply:
Oh, I realized you say `mat3x4`, not `mat4x3`. That makes full sense. I've updated the PR alleviating this case.